### PR TITLE
Added response to review comments

### LINKED ISSUE #40: Create response to reviewers document
We will also want to create a response to reviewers document that describes the changes that you're making. See an example here: https://github.com/greenelab/iscb-diversity-manuscript/blob/master/content/response-to-reviewers.md

### PR DESCRIPTION
**Pull request by danich1**
This PR contains my added response to review comments. Depending on the feedback for this PR some of the changes in #39 may not make sense anymore and I will change as needed (e.g. Hetionet metagraph vs Metagraph subset). 

Side Note: Upcoming PRs will have edited text per declared changes below.

Closes #40

### CODE REVIEW

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
We would like to thank the reviewer for noticing these. We have fixed the typos.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]: I'd suggest not bolding these to make this easier for folks to read.

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]: Should respond to this first paragraph too.

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]: Use methods for general graphs in the first part? Otherwise I don't understand. Also, who considers those graphs to be knowledge graphs? By this point, you should have defined how you are defining knowledge graphs.

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]: Aren't some rephetio edges directional?

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]: This is the point where you're doing the revision process :)

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]: I think they were confused both by the typo but in what the quantities are. Can you take a look at the surrounding sentences and see if you can say things more precisely? I think the high precision / low recall point below might work better for most readers.

File: content/response-to-reviewers.md
Comment (cgreene) [reply content/response-to-reviewers.md]: Also, be consistent about how you bold. I almost missed this.

It looks like it should be: bold your responses, don't bold their questions

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]: Put the text that you are revising here too so your reader doesn't have to go back and forth between the documents.

**Review by cgreene — COMMENTED** [review]
A few thoughts! Let me know when I should take a look through again :)

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md:6]:
```suggestion

**Thank you. We appreciate the positive feedback on our manuscript.

```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**We would like to thank you for noticing these typos. We have fixed each of these during our revision process.**

```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**This point from the reviewer helped us to identify a series of substantial revisions that we needed to make to the manuscript for clarity.
We agree with the reviewer that there is not a single, well-established definition of a knowledge graph in the literature.
Therefore, we should clearly specify a definition and refer to consistent framings in the literature.
We now do this.
For the purposes of this review, we defined a knowledge graph as:
> a resource that integrates one or more expert-derived sources of information into a graph where nodes represent biomedical entities and edges represent relationships between two entities.
This is a broader definition than some.
For example, knowledge graphs that aren't suitable for symbolic reasoning would qualify under this definition.
However, selecting this broader definition allows us to discuss a growing set of methods (including SVD, PCA, node2vec, and other strategies) for producing reduced representations that are then used for prediction.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
We took the reviewer's concerns to heart and now, throughout the manuscript, we clearly specify when transformations produce a representation that is no longer a knowledge graph (e.g., the SVD decomposition or node2vec embeddings) for the purposes of an application.**
```
Please do a read through and make sure we actually do this.

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```
I guess these can't be set apart on their own - at least in the diff it was bolding everything that followed too.

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**For "techniques that represent knowledge graphs in a low dimensional space", we include methods such as node2vec and translational distance models that associate nodes and possibly edges to dense vectors.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
For these techniques, the idea is that these vectors capture the predominant features of a knowledge graph.
This representation can make it easier to apply machine learning and other methods to perform tasks such as link prediction for biomedical applications.
We now clearly specify that these reduced representations are not, themselves, knowledge graphs.**
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**As there was no single, well-accepted definition of a knowledge graph we had to select one from the literature.
We selected a relatively permissive definition because we felt that many recent methodological advances had been demonstrated primarily on more general graphs of biomedical entities extracted from biomedical knowledge bases.
We specifically defined a biomedical knowledge graphs as: "a resource that integrates one or more expert-derived sources of information into a graph where nodes represent biomedical entities and edges represent relationships between two entities."
We agree that not all of these references were about biomedical knowledge graphs.
We have now clarified the first paragraph to discuss _graphs_, before diving into our definition of biomedical knowledge graphs in the second paragraph.**
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```
I don't see any way that a social network is a knowledge graph. I think this is a losing argument. I think the easier fix is just to make the first paragraph be about graphs and then jump into the definition of biomedical knowledge graphs in the second paragraph. Alternatively, you could revise the first one to simply include knowledge graphs.

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**We appreciate the reviewer noticing this.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
As we were fixing this typo, we realized that the sentence itself was unclear and could be improved.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
We updated the text to clarify the point that we intended to make.**
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**We grouped techniques based on the methods that underlie them.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
We agree that the title for this section was overly broad.
We have changed our title to refer to these as representational learning techniques, as this is a term used in the machine learning methods literature for techniques that aim to capture a low-dimensional representation with certain properties.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
Though the output of these methods is not, in and of itself, a knowledge graph, we expect that these methods can be applied to knowledge graphs and have updated text to be more explicit on this point.**
```

File: content/response-to-reviewers.md
Comment (cgreene) [reply content/response-to-reviewers.md]: I filed https://github.com/greenelab/knowledge-graph-review/issues/56 to track this.

**Review by cgreene — COMMENTED** [review]
A few suggestions - I think there's just one more bigger thing to change before submitting this! Woo!

**Comment by danich1** [comment]
Pinging @cgreene to see if changes are approved.

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**We agree with the reviewer that there is not a single, well-established definition of a knowledge graph in the literature.
We identified a series of substantial revisions that we needed to make to the manuscript for clarity.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
We have made revisions to clearly specify a definition and refer to consistent definitions in the prior literature.
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
```

File: content/response-to-reviewers.md
Comment (cgreene) [inline content/response-to-reviewers.md]:
```suggestion
**We agree that not all of these references were about biomedical knowledge graphs.
```

**Review by cgreene — APPROVED** [review]
A couple pretty small changes (though they touch a few lines). After merging, let's get this resubmitted 🚀

**Pull request merged**
